validator/src/lib.rs

```diff
@@ -575,7 +575,6 @@ pub fn main() {
             &node.sockets.gossip,
             &node.sockets.broadcast,
             &node.sockets.repair,
-            &node.sockets.retransmit,
```

Comment:
what was this doing ?

Reply:
Does some check to see if it can reach the socket.

Reply:
Should we give it all the retransmit sockets or is that overkill?

Reply:
It actually seemed to have some kind of problem with it. i'm not sure if there is something with the multi_bind that is incompatible with it. Let me try it again and see.